pygmt/clib/loading.py

```diff
@@ -84,6 +78,37 @@ def clib_name(os_name):
     return libname
```

Comment:
Should really rename all the `libname` to `libnames` (plural) here in the `clib_name` function since it's a list.
```suggestion
    return libnames
```

Reply:
Done in 4bc0bd754783df22b4fc956da6588be671c53e11. I also renamed the function `clib_name` to `clib_names`.